modules/fee-market/src/lib.rs

```diff
@@ -117,8 +116,16 @@ pub mod pallet {
 			Vec<T::AccountId>,
 			Option<T::BlockNumber>,
 		),
-		/// Reward distribute of the order. \[lane_id, message_nonce, rewards\]
-		OrderReward(LaneId, MessageNonce, RewardItem<T::AccountId, BalanceOf<T, I>>),
+		/// Reward distribute of the order. \[lane_id, message_nonce, to_treasury, to_slot_relayer,
+		/// to_message_relayer, to_confirm_relayer\]
+		OrderReward(
+			LaneId,
+			MessageNonce,
+			Option<BalanceOf<T, I>>,
+			Option<(T::AccountId, BalanceOf<T, I>)>,
+			(T::AccountId, BalanceOf<T, I>),
+			BalanceOf<T, I>,
```

Comment:
Here it seems that only amount but no account information for `to_confirm_relayer` ?